Lib/fontTools/varLib/__init__.py

```diff
@@ -60,17 +59,14 @@ def _add_fvar(font, axes, instances):
 	and possibly 'postscriptfontname' entries.
 	"""
 
-	assert axes
-	assert isinstance(axes, OrderedDict)
-
 	log.info("Generating fvar")
 
 	fvar = newTable('fvar')
 	nameTable = font['name']
 
 	for a in axes.values():
 		axis = Axis()
-		axis.axisTag = Tag(a.tag)
+		axis.axisTag = a.tag
```

Comment:
I think it's unnecessary. @anthrotype ?

Reply:
it's unnecessary in this case because sstruct.pack encodes them `tobytes` before. The Tag class is to be able to treat bytes or ascii strings as equivalent, like in py2. I'd suggest you keep it there for now, we can clean up all that py23 legacy in separate PR.